XmlFormatter/src/Windows/PluginManager.cs

```diff
@@ -127,16 +125,16 @@ private void TV_Plugins_AfterSelect(object sender, TreeViewEventArgs e)
                     }
 
 
-                    if (currentPlugin.GetSettingsPage() == null)
-                    {
-                        return;
-                    }
+                    //if (currentPlugin.GetSettingsPage() == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)